templates/custom/flake.nix

```diff
@@ -0,0 +1,48 @@
+{
+  description = "Flake for Holochain app development with customized Holochain build";
+
+  inputs = {
+    holonix.url = "github:holochain/holonix?ref=main";
+
+    nixpkgs.follows = "holonix/nixpkgs";
+    flake-parts.follows = "holonix/flake-parts";
+  };
+
+  outputs = inputs@{ flake-parts, ... }: flake-parts.lib.mkFlake { inherit inputs; } {
+    systems = builtins.attrNames inputs.holonix.devShells;
+    perSystem = { inputs', pkgs, ... }:
+      let
+        # Disable default features and enable CHC (chain head coordination) feature for holochain package.
+        cargoExtraArgs = "--no-default-features --features chc";
```

Comment:
Necessary to disable default here? We should be able to use add the `chc` feature right?

Reply:
Not at all, that was from my `sqlite` feature test.